test/utils.js

```diff
@@ -1,6 +1,13 @@
 const crypto = require("crypto");
 const { Algorithm } = require("@iov/bcp-types");
-const { ethereumCodec } = require("@iov/ethereum");
+const { ethereumCodec, EthereumConnection } = require("@iov/ethereum");
+
+async function getBalance(address) {
+  const connection = await EthereumConnection.establish("http://localhost:7545");
+  const account = await connection.getAccount({ address });
+  const balance = account ? account.balance[0].quantity : "0";
+  return parseInt(balance, 10);
```

Comment:
An Ethereum balanca easily exceeds the safe integer range (1 ETH is 10^18 wei). Thus we can either use `BN` or explicitly use this for rough eastimations: `getBalanceApproximation`, which most likely does the job here